src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,140 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+/**
+ * Represents a class to manage the storing of the TaskList into a text file,
+ * and reading a text file to create a TaskList.
+ */
+public class Storage {
+
+
+    /**
+     * Represents time and date.
+     */
+    private static final SimpleDateFormat DATE_TIME_CONVERTER_FORMAT = new SimpleDateFormat("MMM dd yyyy hh:mma");
+    /**
+     * Represents date.
+     */
+    private static final SimpleDateFormat DATE_CONVERTER_FORMAT = new SimpleDateFormat("MMM dd yyyy");
+
+
+    /**
+     * File path where the text file is stored.
+     */
+    private Path filepath;
+
+
+    /**
+     * Initialises Storage with filepath where text file is stored.
+     *
+     * @param filepath Filepath where the text file is stored.
+     */
+    public Storage(String filepath) {
+        this.filepath = Paths.get(System.getProperty("user.dir") + filepath);
```

Comment:
Perhaps this is unnecessary? The basic coding standard doesn't require this, but is suggested by the advanced style guide.
```suggestion
        filepath = Paths.get(System.getProperty("user.dir") + filepath);
```

Reply:
I noticed this in several other places too.